gtfs.go

```diff
@@ -106,3 +106,44 @@ func loadGTFS(g *GTFS, filter map[string]bool) error {
 	}
 	return nil
 }
+
+// Dump GTFS data to an already existing directory
+// @param g: GTFS struct to dump
+// @param dirPath: Target directory
+// @param filter: same as for load function
+// @return error
+func Dump(g *GTFS, dirPath string, filter map[string]bool) error {
+    _, err := os.Stat(dirPath)
+    if os.IsNotExist(err) {
+        return fmt.Errorf("Error dumping GTFS: target directory doesn't exist")
+    }
+
+    agencySlice := make([]Agency, 1)
+    agencySlice[0] = g.Agency
+
+    files := map[string]interface{}{
+        "agency.txt":         agencySlice,
```

Comment:
I think you could have done that: `[]Agency{g.Agency}` to limit number of lines, but it has been a will since I wrote any go code so I might be wrong.